src/__tests__/flex.spec.ts

```diff
@@ -12,6 +12,10 @@ describe(`flex grow/shrink`, () => {
     [`flex-grow-0`, { flexGrow: 0 }],
     [`flex-grow`, { flexGrow: 1 }],
     [`flex-grow-77`, { flexGrow: 77 }],
+    [`grow`, { flexGrow: 1 }],
+    [`grow-0`, { flexGrow: 0 }],
+    [`shrink`, { flexShrink: 1 }],
+    [`shrink-0`, { flexShrink: 0}],
```

Comment:
can you add a couple of tests to show that arbitrary grow/shrink values work as well? like `grow-33` and `shrink-77`?

Reply:
thanks, added some more test